libraries/AP_Scheduler/AP_Scheduler.h

```diff
@@ -131,3 +132,5 @@ class AP_Scheduler
     // performance counters
     AP_HAL::Util::perf_counter_t *_perf_counters;
 };
+
+#include "AP_Scheduler.tpp"
```

Comment:
@lucasdemarchi 
Header only is necessary to avoid instantiation, which has only disadvantages if someone adds a new example or new vehicle types.